_typos.toml

```
@@ -4,6 +4,7 @@
 [default.extend-identifiers]
 
 [default.extend-words]
+ba="ba"
```

調べた感じ辞書にない単語を無視するための項目だと思いますが、`ba`という単語どこかにありましたっけ...

- <https://github.com/crate-ci/typos/blob/ad88239051ada08a9976d84c698f5661d989ebe5/docs/reference.md>

---

`build/installer_linux.sh` の `-ba`
```sh
APPIMAGE=$(${COMMAND_7Z} l -slt -ba "${FIRST_ARCHIVE}" | grep 'Path = ' | head -n1 | sed 's/Path = \(.*\)/\1/')
```
に引っかかってしまうので除外しています

---

なるほどです！ p7zipの7zコマンドのオプション引数が引っかかっているのですね...

p7zip: <https://sourceforge.net/projects/p7zip/files/p7zip/>

---

なるほどです･･･。
意図をコメントに残しておくと、不要になった際に消す判断がしやすくなるかもと思いました。

こんな感じでどうでしょう。
```suggestion
ba="ba" # 7zコマンドの-baオプション
```